drivers/bmx280/bmx280.c

```diff
@@ -50,10 +50,7 @@
 #ifdef BMX280_USE_SPI /* using SPI mode */
 static inline int _acquire(const bmx280_t *dev)
 {
-    if (spi_acquire(BUS, CS, MODE, CLK) != SPI_OK) {
-        return BMX280_ERR_BUS;
-    }
-    return BMX280_OK;
+    spi_acquire(BUS, CS, MODE, CLK);
 }
```

Comment:
Either let `_acquire` returns void or re-add `return BMX280_OK` for consistency

Reply:
Yes indeed, didn't pay close enough attention. I reverted ca32f97d210d671e1e379d27f58564180e66fcd8